torch/ao/quantization/quantization_mappings.py

```diff
@@ -102,6 +103,7 @@
 DEFAULT_DYNAMIC_QUANT_MODULE_MAPPINGS : Dict[Callable, Any] = {
     nn.GRUCell: nnqd.GRUCell,
     nn.Linear: nnqd.Linear,
+    nnqatd.Linear: nnqd.Linear,
```

Comment:
I just followed the description of T99702853 for this line, but I'm not sure if this does anything. I believe this is only used when we call `quantize_dynamic`, which doesn't currently support QAT.

Reply:
Yeah but its good to list it here, its a mapping so you can think of it as a general list of what modules map to which other modules, even if the current implementation doesn't use it.